bundles/org.openhab.binding.dreamscreen/src/main/java/org/openhab/binding/dreamscreen/internal/message/RefreshMessage.java

```diff
@@ -28,46 +28,68 @@
     static final byte COMMAND_UPPER = 0x01;
     static final byte COMMAND_LOWER = 0x0A;
 
-    protected RefreshMessage(final byte[] data, final int off) {
-        super(data, off);
-    }
+    // Set integers so we know which bytes to pull based on device
+
+    private int groupInt = 32;
```

Comment:
make them static final int and constant names in upper case, (required to pass the review; all of them) e.g.
private static final int OFF_GROUP_ID = 32;